src/types.ts

```diff
@@ -65,10 +66,53 @@ export interface ServerOptions {
    */
   reusePort?: boolean;
 
+  /**
+   * HTTPS configuration options.
+   * If provided, the server will use HTTPS instead of HTTP.
+   * You need to provide at minimum the key and cert options.
+   */
+  https?: {
```

Comment:
I would prefer to have a boolean flag like `enabled` here to easily disable `https` in development env.

If this is set to `true` an error should be thrown instead of falling back to `http` when no certificates are provided.

Reply:
I'm not sure about the `enabled` flag at all.

Reply:
I agree that we need a way to disable TLS directly. even in playground it is needed!

@OskarLebuda, What alternatives do you suggest?

Reply:
We talked to introduce a new `type: "https" | "https"` option that can be extendable to more types like `http2`